editions/tw5.com/tiddlers/community/resources/TiddlyWiki5 Squared by Iannis Zannos.tid

```diff
@@ -15,4 +15,6 @@ The Node.js implementation in TiddlyWiki5 adds all the advantages of flat-file m
 Also very cool is the treatment of tags as menus everywhere.
 <<<
 
-[ext[https://github.com/iani/tw5square]] (download and save index.html and open it in your browser).
+<a href={{!!url}}><$text text="http://larigot.avarts.ionio.gr/users/iani/wikis/tw5square.html"/></a>
```

Comment:
it seems, this link doesn't work!

Reply:
The link's URL is in the `{{!!url}}` field : https://web.archive.org/web/20170728212414/http://larigot.avarts.ionio.gr/users/iani/wikis/tw5square.html

By doing it this way, it shows where the original link used to be, but since that is no longer available, it opens an `archive.com` copy of it.

If that's not needed, then I can simply change the display portion to the `archive.com` page - but that is one long ugly link to display on the page.

Reply:
ahh ... I see. If it opens the archive, if you click the link it would be ok for me.